### DISCUSSION TITLE
Diary-粤了（三） | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#342] [Announcements]
# Diary-粤了（三） | Zi-Zi's Journey

东边有莞草！

https://gz-metal-cell.github.io/2024/07/25/Diary-%E7%B2%A4%E4%BA%86%EF%BC%88%E4%B8%89%EF%BC%89/

**Comment by promefire** [comment]
罂粟花确实好看

**Reply by GZ-Metal-Cell** [reply]
😭pro 哥看完了近代中国整个禁毒历程后得出了结论：罂粟花确实好看！
